Cargo.toml

```diff
@@ -1,6 +1,88 @@
+# https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#workspace-inheritance
+cargo-features = ["workspace-inheritance"]
+
 [workspace]
-members = ["contracts/*", "packages/*", "debug/*", "ci/*"]
+members = [
+	"contracts/cw-dao-core",
+	"contracts/voting-modules/*",
+	"contracts/proposal-modules/*",
+	"contracts/staking-rewards/*",
+	"contracts/utils/*",
+	"packages/*",
+	"debug/*",
+	"ci/*"
+	]
+
+[workspace.package]
+version = "2.0.0"
+edition = "2021"
+license = "Apache-2.0"
+repository = "https://github.com/DA0-DA0/dao-contracts"
+
+[workspace.dependencies]
+# Packages.
+cw-dao-core-interface = { path = "packages/cw-dao-core-interface", version = "*" }
+cw-dao-core-macros = { path = "packages/cw-dao-core-macros", version = "*" }
+cw-paginate = { path = "packages/cw-paginate", version = "*" }
+cw721-controllers = { path = "packages/cw721-controllers", version = "*" }
+indexable-hooks = { path = "packages/indexable-hooks", version = "*" }
```

Comment:
Do we want to prefix some of these packages with `cw`?

Note sure if `voting` will be available on crates. :joy:

Reply:
Great point. :) How about:

`voting` -> `cw-dao-voting`
`indexable-hooks` -> `cw-indexable-hooks`
`proposal-hooks` -> `cw-dao-proposal-hooks`
`vote-hooks` -> `cw-dao-vote-hooks`
`proposal-module-testing` -> `cw-dao-proposal-module-testing`

Sets some pattern were we're doing `cw-` when it's CosmWasm relevant and `cw-dao` if it's DAO DAO relevant.